habitat/hooks/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cd /hab/svc/dcob
+
+export RACK_ENV=production
+export SSL_CERT_FILE="$(hab pkg path core/cacerts)/ssl/certs/cacert.pem"
```

Comment:
Gonna roll this back to a `pkg_svc_run` command in the plan and make setting these a requirement prior to running `hab start chef/dcob`.